challenges/2021_cloud_base_height/03_pytorch_train.ipynb

```diff
@@ -0,0 +1,1039 @@
+{
```

Comment:
<p>is this specific to your model, or could this logger be reused for pytorch lightning generally?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
I've overwritten the class as (I think this was a suggestion to someone else's issue in coding Q&A forum), it was an attempt to overcome/debug the autologging errors. I've written about this in the markdown though for clarity